FreeRTOSv202112.00/FreeRTOS/Demo/ARM7_LPC2129_Keil_RVDS/Task1 Files/source/task2.c

```diff
@@ -0,0 +1,67 @@
+#include "task2.h"
+#include "LED.h"
+
+
+TaskHandle_t Toggle_1000_TaskHandle = NULL;					// Toggle 1000 ms Task Handler
+TaskHandle_t Toggle_500_TaskHandle = NULL;					// Toggle 500 ms Task Handler 
+TaskHandle_t Toggle_100_TaskHandle = NULL;					// Toggle 100 ms Task Handler
+
+// A task to toggle a LED every 1000 ms 
+void vToggle1000Task(void* pvParameters)
+{
+	for(;;)
+	{
+		led_toggle(2,1000);
```

Comment:
You shouldn't use magic numbers

Reply:
I resolved it

Reply:
https://github.com/MahmoudMatarawy/RTOS/blob/Task_1/FreeRTOSv202112.00/FreeRTOS/Demo/ARM7_LPC2129_Keil_RVDS/Task1%20Files/source/task2.c